packages/fxa-auth-server/lib/payments/google-play/play-billing.ts

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Firestore } from '@google-cloud/firestore';
+import { google } from 'googleapis';
+import { Container } from 'typedi';
+import { TypedCollectionReference } from 'typesafe-node-firestore';
+
+import { AppConfig, AuthFirestore, AuthLogger } from '../../types';
+import { PurchaseManager } from './purchase-manager';
+import { IapConfig } from './types';
+import { UserManager } from './user-manager';
+
+export class PlayBilling {
+  private firestore: Firestore;
+  private log: AuthLogger;
+  private prefix: string;
+  private iapConfigDbRef: TypedCollectionReference<IapConfig>;
+
+  public purchaseManager: PurchaseManager;
+  public userManager: UserManager;
+
+  constructor() {
+    const config = Container.get(AppConfig);
+    this.prefix = `${config.authFirestore.prefix}googleIap`;
```

Comment:
do we want any kind of character delimiter between parts? sometimes they're useful when grepping

Reply:
The prefix stored ends in a `-`, I ended up keeping it this way to match the event-broker style.